skimage/exposure/exposure.py

```diff
@@ -499,6 +501,10 @@ def is_low_contrast(image, fraction_threshold=0.05, lower_percentile=1,
     True
     >>> is_low_contrast(image, upper_percentile=100)
     False
+
+    See Also
```

Comment:
See also is reserved by sphinx for referring to another place in the *code*.  You'll have to make this a `References` section.